crates/noirc_evaluator/src/ssa_refactor.rs

```diff
@@ -0,0 +1,6 @@
+//! SSA stands for Single Static Assignment
+//! This module will convert the monomorphized AST
+//! into an SSA IR. This IR will consist of basic blocks and
+//! a control flow graph.
```

Comment:
We should probably put the new M-Ast -> New-SSA pass in its own file like how we have the ssagen file now, but it can be done in a later PR when the pass is actually added as well.

Reply:
Good catch -- I've redone that comment and added more stub types. 

I took away some of the struct definitions I had on the call and will create issues for those instead to fill them in